src/Evolution/Systems/GrMhd/GhValenciaDivClean/BoundaryConditions/BoundaryCondition.hpp

```diff
@@ -25,7 +25,11 @@ class ProductOfConditions;
 namespace detail {
 template <typename DerivedGhCondition, typename DerivedValenciaCondition>
 using ProductOfConditionsIfConsistent = tmpl::conditional_t<
-    DerivedGhCondition::bc_type == DerivedValenciaCondition::bc_type,
+    (DerivedGhCondition::bc_type !=
```

Comment:
isn't this just `not (DerivedGhCondition::bc_type ==  evolution::BoundaryConditions::Type::Outflow xor DerivedValenciaCondition::bc_type == evolution::BoundaryConditions::Type::Outflow)` in which case you can get rid of the `not` by reversing the order of the next arguments of the conditonal_t